pkg/network/node/ovs/ovs.go

```diff
@@ -424,9 +425,19 @@ func (tx *ovsExecTx) DeleteFlows(flow string, args ...interface{}) {
 }
 
 func (tx *ovsExecTx) Commit() error {
-	err := tx.ovsif.bundle(tx.flows)
-
-	// Reset flow context
-	tx.flows = []string{}
-	return err
+	defer func() {
+		tx.flows = []string{}
+	}()
+	retries := 30
+	for {
```

Comment:
The idea is good, but I'd use `utilwait.ExponentialBackoff` instead of doing this by hand.